changelog.xml

```diff
@@ -505,4 +504,19 @@
 			<dropTable tableName="ExperimentImport" />
 		</rollback>
 	</changeSet>
+
+	<changeSet id="experiment-template-id" author="nicolaikrebs">
+		<preConditions onFail="MARK_RAN">
+			<tableExists tableName="Experiment" />
+		</preConditions>
+		<addColumn tableName="Experiment">
+			<column name="templateId" type="varchar(500)">
```

Comment:
500 is a bit big for storing integer ids serialized as strings. Anything between 50 and 255 should work just fine (the larger values are only interesting if we want to use text ids for templates in the future but even then 50 is already plenty).